h/static/scripts/controllers/lozenge-controller.js

```diff
@@ -18,9 +19,21 @@ class LozengeController extends Controller {
   constructor(containerEl, options) {
     super(containerEl, options);
     var lozengeEl = document.createElement('div');
+    var lozengeMakeup = options.content;
+
+    if(SearchTextParser.hasKnownNamedQueryTerm(options.content)) {
+      var queryTerm = SearchTextParser.getLozengeFacetNameAndValue(options.content);
+      lozengeMakeup = '<span class="lozenge__facet-name">' +
```

Comment:
Injecting user-controlled content into HTML markup enables trivial XSS attacks: [See this demo](http://localhost:5000/search?q=tag%3A%22%3Cimg%20src=%27http://i.giphy.com/TOWeGr70V2R1K.gif%27%3E%22+)